crates/re_types_builder/src/codegen/docs/mod.rs

```diff
@@ -84,10 +83,9 @@ impl CodeGenerator for DocsCodeGenerator {
     }
 }
 
-fn frontmatter(o: &mut String, title: &str, order: u32) {
+fn frontmatter(o: &mut String, title: &str) {
```

Comment:
Arguably we could have still used the order for determining what comes first on the website 🤔

Reply:
That would be a very bad way to decide the order imho - randomly scattered integers.

And no, let's use lexicographical order instead. Simple, clean, elegant.